### DISCUSSION TITLE
SSR Mode Fails on Deploy When Importing Nested Files Inside Node Module

### DISCUSSION THREAD

**Discussion by 1b4Pi** [quasarframework/quasar#12424] [CLI - SSR mode]
I've got a Quasar SSR/PWA handoff web app that crashes on the server with the catch-all 500 error as soon as I try to import js files bundled within an installed node_module dependency. 

Everything works fine in both dev env (spa) and prod env (ssr) when using deps from:
`import { Vector2 } from "three";`

But prod fails when trying to:
`import { EffectComposer } from "three/examples/jsm/postprocessing/EffectComposer";`

I tried including the js file in my boot config:

`boot: [
    "i18n",
    "axios",
    ...
    "~three/examples/jsm/postprocessing/EffectComposer",
],`

That didn't resolve the issue on its own. I feel like I'm missing a step here but it's not clear in the docs.

I also tried importing from a boot file (properly?):

src/boot/composer.js:

```
import { boot } from "quasar/wrappers";
import { EffectComposer } from "three/examples/jsm/postprocessing/EffectComposer";

export default boot(({ app }) => {});
export { EffectComposer };
```

quasar.conf.js:

`boot: [
    "i18n",
    "axios",
    ...
    "composer",
],`

And finally the Vue file:

`import { EffectComposer } from "../../../../boot/composer";`

While everything is fine in SPA dev land. Server-side this is what I'm getting on the logs:

```
[Vue Router warn]: uncaught error during route navigation:
ReferenceError: Cannot access '__WEBPACK_DEFAULT_EXPORT__' before initialization at Module.default (chunk-754.js:36:42) at /home/cleavr/thr9cpwmnwg8dodd3518.cleaver.rocks/releases/20220209085715950/node_modules/vue-router/dist/vue-router.cjs.js:2017:36
```

Because I'm working with scripts that use WebGL, I thought maybe that was the issue and went down a wild path of finding a VPS that offers Ubuntu20x with NVIDIA GPU support. While it was fun spinning that server up... alas, that damn 500 error still showed on deployment with both the GPU and CPU servers.

Any pointers as to what I'm doing wrong or missing? Many thanks. Quasar is very, very, rad.

**Comment by suleimanas** [comment]
just for test: Try
`import { EffectComposer } from "app/node_modules/three/examples/jsm/postprocessing/EffectComposer";`
